docs/standard/data/sqlite/connection-strings.md

```diff
@@ -86,6 +86,15 @@ A value that indicates whether to enable recursive triggers.
 | True  | Sends `PRAGMA recursive_triggers` immediately after opening the connection. |
 | False | Doesn't send `PRAGMA recursive_triggers`. This is the default.              |
 
+### Default Timeout
+
+The default timeout (in seconds) for executing commands. The default value is 30. *Command Timeout* is an alias of this keyword.
```

Comment:
```suggestion
The default timeout (in seconds) for executing commands. The default value is 30. The*command timeout* is an alias of this keyword.
```

I believe that we're confusing the way to show these keywords, or aliases, or property names. If we're referring to it conceptually, we should call it "default timeout", or as it's property name `DefaultTimeout`. It shouldn't be *Default Timeout".

Reply:
No properties here. Connection strings only have keywords. Some keywords (like `Command Timeout`) are merely aliases for the canonical form of the keyword (in this case, `Default Timeout`)

Reply:
Ok, in that case we need to add:

```markdown
no-loc: Command Timeout, Default Timeout, Data Source, etc. 
```

To the docs' metadata, it wasn't obvious that these were keywords.